lib/active_record_view_model/update_data.rb

```diff
@@ -22,6 +46,18 @@ def has_key?(name)
       end
     end
 
+    def self.describe_json_schema_error(error)
+      "Error parsing #{error.schema.description}: #{error.message}"
+    end
+
+    def self.verify_schema!(schema, value)
+      valid, errors = schema.validate(value)
+      unless valid
+        an_error = errors.first
```

Comment:
Can return a human only readable version of the errors, as discussed, but we should revisit this when we redo the errors for path information.